2.13/tests/test/test.js

```diff
@@ -473,10 +447,57 @@ function testDeprovision(instanceId, queryStrings, isAsync){
                     done();
                 })
             });
+
+            if (isAsync) {
+                // TODO: the query string should contain 'operation' for the last operation. FYI: https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#parameters.
+                describe('DEPROVISION - poll', function() {
+                    it ('should return succeeded operation status after deprovision', function(done){
+                        pollInstanceLastOperationStatus(instanceId, done);

```

Comment:
@leonwanghui as you can see, the default timeout for each case is 2000ms in mocha, have a try `this.timeout(timeOutInMs)`. Ref: https://mochajs.org/#timeouts

Reply:
And it should be also applied in provision-poll.